doc/Changelog.md

```diff
@@ -2,7 +2,8 @@ Changelog {#Changelog}
 =========
 
 # git master
-
+* [138](https://github.com/BlueBrain/Brion/pull/138):
+  brain::CompartmentReport : python API v2
```

Comment:
This changelog registers relevant changes between releases. The intra-release history is not relevant. That means that I'd remove this Changelog entry, pick 130, move it to the top and rewrite it as:

    * [130](https://github.com/BlueBrain/Brion/pull/130), [138](https://github.com/BlueBrain/Brion/pull/138):
      Added brain::CompartmentReport and its Python wrapping.